app/views/devise/confirmations/new.html.erb

```diff
@@ -1,22 +1,20 @@
 <div class="main__content profile">
-  <div class="main__scroll pad-all">
-    <h1 class="h2">Resend confirmation instructions</h1>
+  <h1 class="h2">Resend confirmation instructions</h1>
 
-    <%= simple_form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
-      <%= f.error_notification %>
-      <%= f.full_error :confirmation_token %>
+  <%= simple_form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+    <%= f.error_notification %>
+    <%= f.full_error :confirmation_token %>
 
-      <div class="form-inputs">
-        <%= f.input :email, input_html: { class: "form-control--text" }, required: true, autofocus: true %>
-      </div>
-
-      <div class="form-actions">
-        <%= f.button :submit, "Resend instructions", class: "btn btn-primary" %>
-      </div>
-    <% end %>
+    <div class="form-inputs">
+      <%= f.input :email, input_html: { class: "form-control--text" }, required: true, autofocus: true %>
+    </div>
 
-    <div class="margin-top-double">
-      <%= render "devise/shared/links" %>
+    <div class="form-actions">
+      <%= f.button :submit, "Resend instructions", class: "btn btn-primary" %>
     </div>
+  <% end %>
+
+  <div class="margin-top-double">
```

Comment:
Change this class (these type classes) to use `--` in the source. Update everywhere used